filebeat/tests/system/test_modules.py

```diff
@@ -176,7 +176,7 @@ def _test_expected_events(self, test_file, objects):
                 clean_keys(obj)
 
                 # Remove timestamp for comparison where timestamp is not part of the log line
-                if obj["fileset.module"] == "icinga" and obj["fileset.name"] == "startup":
+                if obj["event.module"] == "icinga" and obj["event.dataset"] == "startup":
```

Comment:
I've seen that hack too, I think we could add a test.json inside the module to configure the assertions, to do in another PR of course.

Reply:
yes, would be nice to do it differently.